package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.13.1",
     "minimist": "^1.1.1",
     "request": "^2.69.0",
+    "v8-compile-cache": "^1.1.0",
```

Comment:
Small point, but could we have this in `devDependencies` instead? Since it's only used in the gulpfile, we only need it at build-time.